pandas/conftest.py

```diff
@@ -401,13 +407,36 @@ def string_dtype(request):
     return request.param
 
 
+@pytest.fixture(params=OBJECT_DTYPES)
+def object_dtype(request):
+    """Parametrized fixture for object dtypes.
+
+    * object
+    * 'object'
+    """
+    return request.param
+
+
+@pytest.fixture(params=TIME_DTYPES)
+def time_dtype(request):
```

Comment:
this is not a good fixture name, nor correct as these are not times but datetimes and timedeltas which are very separate